rust/parser/pipeline.rs

```diff
@@ -235,6 +237,24 @@ fn visit_statement_deletable(node: Node<'_>) -> Deletable {
     Deletable::new(span, kind)
 }
 
+fn visit_pattern_try_deletable(node: Node<'_>) -> Deletable {
+    debug_assert_eq!(node.as_rule(), Rule::pattern_try_deletable);
+    let span = node.span();
+    let deletables = node
+        .into_children()
+        .skip_expected(Rule::TRY)
+        .map(|child| match child.as_rule() {
+            Rule::statement_deletable => visit_statement_deletable(child),
+            Rule::pattern_try_deletable => visit_pattern_try_deletable(child),
```

Comment:
The grammar doesn't allow a `delete { try... { try { ... }}` does it? so why is this here?